layers/+tools/lsp/funcs.el

```diff
@@ -0,0 +1,18 @@
+(defun lsp//sync-peek-face ()
```

Comment:
What does this function do exactly ? Can you add a docstring ?

Reply:
Done